fele-client-service/client-api/scripts/channel.js

```diff
@@ -0,0 +1,28 @@
+const { checkIfNetworkExists, insertToDatabase } = require('../../utils/db')
+const path = require("path");
+const fs = require('fs');
+
+const createChannel = async (networkName, channelName, channelConfigJSON) => {
+    const database = "fele__"+networkName;
```

Comment:
Can we make this a function so that we can use these in multiple apis or cli also, like some function that returns the naming so that it's common

Reply:
We can add a constants.js file to add all constants. And add this prefix to the constants. function would be not that good